src/main/java/com/pontusvision/graphutils/gdpr.groovy

```diff
@@ -5819,9 +5828,11 @@ the end of the process.
         e.printStackTrace()
       }
 
-      App.g.V()
-              .has('Metadata_Type_Event_Subject_Access_Request', eq('Event_Subject_Access_Request'))
-              .groupCount().by('Event_Subject_Access_Request_Status')
+      App.graph.executeSql(

```

Comment:
Você pode tentar criar alguns testes, please?  Acho que está ficando ótimo, mas só para termos certeza

Reply:
on it! 💪